# HG changeset patch
# User jeremy avnet <spork-bitbuc...@theory.org>
# Date 1266954792 18000
# Node ID 03897731025dafe19ea8ed6b452f312391dd9d86
# Parent  5864df9bf64d63dc0d43653ca66094daca339f42
findoutgoing can return either a tuple or a single value, handle both

diff --git a/hggit/hgrepo.py b/hggit/hgrepo.py
--- a/hggit/hgrepo.py
+++ b/hggit/hgrepo.py
@@ -24,8 +24,11 @@
             if isinstance(remote, gitrepo):
                 git = GitHandler(self, self.ui)
                 base, heads = git.get_refs(remote.path)
-                out, h = super(hgrepo, self).findoutgoing(remote, base, heads, force)
-                return out
+                retval = super(hgrepo, self).findoutgoing(remote, base, heads, force)
+                if isinstance(retval, tuple):
+                    return retval[0]
+                else:
+                    return retval
             else: #pragma: no cover
                 return super(hgrepo, self).findoutgoing(remote, base, heads, force) 
